tg_bot/__init__.py

```diff
@@ -83,6 +91,14 @@
         WHITELIST_USERS = set(int(x) for x in Config.WHITELIST_USERS or [])
     except ValueError:
         raise Exception("Your whitelisted users list does not contain valid integers.")
+    try:
+        WHITELIST_CHATS = set(int(x) for x in Config.WHITELIST_CHATS or [])
+    except ValueError:
+        raise Exception("Your whitelisted users list does not contain valid integers.")
+    try:
+        BLACKLIST_CHATS = set(int(x) for x in Config.BLACKLIST_CHATS or [])
+    except ValueError:
+        raise Exception("Your whitelisted users list does not contain valid integers.")
```

Comment:
same as the other 3

Reply:
The exception strings have been taken from the WHITELIST exception strings and only the values where edited